packages/yoroi-extension/features/step_definitions/common-steps.js

```diff
@@ -186,32 +196,34 @@ function createDirInTestRunsData(subdirectoryName) {
 
 async function takeScreenshot(driver, name) {
   // path logic
-  const dir = createDirInTestRunsData('screenshots');
+  const dir = await createDirInTestRunsData(driver, 'screenshots');
   const path = `${dir}/${testProgress.step}_${testProgress.lineNum}-${name}.png`;
 
   const screenshot = await driver.takeScreenshot();
   await writeFile(path, screenshot, 'base64');
 }
 
 async function takePageSnapshot(driver, name) {
-  const dir = createDirInTestRunsData('pagesSnapshots');
+  const dir = await createDirInTestRunsData(driver, 'pagesSnapshots');
   const htmlLogPath = `${dir}/${testProgress.step}_${testProgress.lineNum}-${name}-dom.html`;
   const html = await driver.executeScript('return document.body.innerHTML;');
   await fsAsync.writeFile(htmlLogPath, html);
 }
 
+async function getDriverLogs(driver, name) {
+  const dir = await createDirInTestRunsData(driver, 'driverLogs');
+  const consoleLogPath = `${dir}/${testProgress.step}_${testProgress.lineNum}-${name}-driver-log.json`;
+  const logEntries = await driver.manage().logs().get(logging.Type.DRIVER);
+  const jsonLogs = logEntries.map(l => l.toJSON());
+  await fsAsync.writeFile(consoleLogPath, JSON.stringify(jsonLogs));
+}
+
 async function getConsoleLogs(driver, name) {
-  const cap = await driver.getCapabilities();
-  const browserName = cap.getBrowserName();
-  if (browserName === 'firefox'){
-    return;
-  }
-  const dir = createDirInTestRunsData('consoleLogs');
+  const dir = await createDirInTestRunsData(driver, 'consoleLogs');
   const consoleLogPath = `${dir}/${testProgress.step}_${testProgress.lineNum}-${name}-console-log.json`;
   const logEntries = await driver.manage().logs().get(logging.Type.BROWSER);
   const jsonLogs = logEntries.map(l => l.toJSON());
   await fsAsync.writeFile(consoleLogPath, JSON.stringify(jsonLogs));
-
 }
```

Comment:
These two functions are almost identical. The difference are in the logs name and the logging type. So maybe is better to create one common function with conditions?